cfbs/git_magic.py

```diff
@@ -4,12 +4,16 @@
 do prompts, etc.
 """
 
+from collections import namedtuple
 from cfbs.prompts import YES_NO_CHOICES, prompt_user
 from cfbs.cfbs_config import CFBSConfig, CFBSReturnWithoutCommit
 from cfbs.git import git_commit, git_discard_changes_in_file, CFBSGitError
 from functools import partial
 
 
+Result = namedtuple("Result", ["rc", "commit", "msg_body"])
```

Comment:
I don't like that you can only set the body, but not the title (first line) of the commit message in the result. I'd want to be able to create different commit messages (including title) depending on what happened in the function.

Reply:
Now you override the entire commit message if 
```Python
msg is not None
```